### PR TITLE
feat: add expand all

### PR DESCRIPTION
**Pull request by songwongtp**
## Describe your changes
- Add expand/collapse all for unsupported pool list

### CODE REVIEW

**Review by bkioshn — APPROVED** [review]

File: src/lib/pages/pools/components/unsupportedSection/index.tsx [hunk 1]
```diff
@@ -158,13 +161,27 @@ export const UnsupportedSection = ({
...
+                )
+              }
+            >
+              {expandedIndex.length ? "Collapse All" : "Expand All"}
             </Button>
           </Flex>
         </Flex>
       </Flex>
-      <UnsupportedPoolList pools={pools} isLoading={isLoading} />
+      <UnsupportedPoolList
```
Comment (evilpeach) [inline src/lib/pages/pools/components/unsupportedSection/index.tsx:179]: To reduce the complexity of type involved, I prefer to pass a function that wraps `setExpandedIndex`, like this.
```tsx
  const updateExpandedIndex = (indexes: number[]) => {
    setExpandedIndex(indexes);
  };
```
and pass as props on `UnsupportedPoolList`
```tsx
setExpandedIndex={updateExpandedIndex}
```

File: src/lib/pages/pools/components/unsupportedSection/UnsupportedPoolList.tsx [hunk 2]
```diff
@@ -9,11 +10,15 @@ import { UnsupportedPoolCard } from "./UnsupportedPoolCard";
 interface UnsupportedPoolListProps {
   pools: Option<Pool[]>;
   isLoading: boolean;
+  expandedIndex: number[];
+  setExpandedIndex: Dispatch<SetStateAction<number[]>>;
```
Comment (evilpeach) [inline src/lib/pages/pools/components/unsupportedSection/UnsupportedPoolList.tsx:14]: So, you can change this type to `(indexes: number[]) => void`.

File: src/lib/pages/pools/components/unsupportedSection/UnsupportedPoolList.tsx [hunk 3]
```diff
@@ -27,7 +32,11 @@ export const UnsupportedPoolList = ({
 
   return (
     <SimpleGrid columns={1} spacing={4} w="full">
-      <Accordion allowMultiple>
+      <Accordion
+        allowMultiple
+        index={expandedIndex}
+        onChange={(index) => setExpandedIndex(index as number[])}
```
Comment (evilpeach) [inline src/lib/pages/pools/components/unsupportedSection/UnsupportedPoolList.tsx:38]: and change this to `onChange={setExpandedIndex}`

**Review by evilpeach — CHANGES_REQUESTED** [review]
`updateExpandedIndex` is an example name. You can change what you like.

**Review by evilpeach — APPROVED** [review]

**Pull request merged**